src/main/java/redis/clients/jedis/CommandObjects.java

```diff
@@ -3836,7 +3836,7 @@ public final CommandObject<Boolean> bfAdd(String key, String item) {
 
   public final CommandObject<List<Boolean>> bfMAdd(String key, String... items) {
     return new CommandObject<>(commandArguments(BloomFilterCommand.MADD).key(key).
-        addObjects((Object[]) items), BuilderFactory.BOOLEAN_LIST);
+        addObjects((Object[]) items), BuilderFactory.BOOLEAN_WITH_ERROR_LIST);
```

Comment:
FYI I'm working on he RESP3 bits with the module teams still. I wouldn't put effort into RESP3 for modules yet.

Reply:
This is not RESP3 specific change.